account_invoice_report_commercial_address/reports/report_invoice.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2025 Ángel Rivas <angel.rivas@sygel.es>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template
+        id="account_invoice_report_commercial_address_report_invoice_document"
+        name="account.invoice.report.commercial.address.report.invoice.document"
+        inherit_id="account.report_invoice_document"
+        priority="999"
+    >
+            <xpath expr="//div[@id='total']" position="inside">
+                <div class="commercial-address position-absolute start-0 ms-3">
+                    <t t-if="o.show_commercial_entity">
```

Comment:
Have you tried using the t-options with the contact widget to show the information of the address?

Reply:
In that case, you can leave it as it was before. Sorry about that.